pkg/ndctl/fake/region.go

```diff
@@ -105,6 +105,10 @@ func (r *Region) SeedNamespace() ndctl.Namespace {
 	return nil
 }
 
+func (r *Region) GetAlign() uint64 {
+	return uint64(mib * 16)
```

Comment:
This should better be a member variable to enable unit testing with different alignments.